data/test-cases/plugins/MS Teams sync/Slash commands/Connect.md

```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "Connect user slash command"
+status: Active
+priority: Normal
+folder: Slash commands
+authors: "@AayushChaudhary0001"
+team_ownership: 
+- Change Team Name
```

Comment:
What is this?

Reply:
Slash commands is the name of the folder, and we do not write the team name